app/engines/bing_search.rb

```diff
@@ -12,7 +12,6 @@ def initialize(options = { })
     super
     @options = options
     @api_connection = self.class.api_connection
-    Rails.logger.error("CACHE_LIFETIME: #{CACHE_LIFETIME}")
```

Comment:
This is only giving us false alarms. No need to log this, and no need to log it as an error.